catris/squares.py

```diff
@@ -20,18 +20,17 @@ class _RotateMode(Enum):
 
 class Square:
     def __init__(self, rotate_mode: _RotateMode) -> None:
-        self.x = 0
-        self.y = 0
-        # The offset is a vector from current position (x, y) to center of rotation
+        # Where is the square when it's created? Relative to rotation center.
+        self.original_x = 0
+        self.original_y = 0
```

Comment:
These are always same as `original_offset_x` and `original_offset_y`, but with different signs. Should get rid of one of the two